.github/workflows/tests.yaml

```
@@ -142,7 +142,7 @@ jobs:
       - name: Install additional test dependencies
         run: |
 
-          sudo apt install -y ninja-build cmake scons qt5-qmake \
+          sudo apt install -y ninja-build cmake scons qt5-qmake p7zip-full \
```

```suggestion
          sudo apt install -y ninja-build cmake scons qt5-qmake p7zip \
```

Let's ensure this works with just a basic 7-zip

---

Done